components/CreateVirtualCardsForm.js

```diff
@@ -392,23 +394,37 @@ class CreateVirtualCardsForm extends Component {
   }
 
   renderManualFields() {
+    const { collectiveSettings } = this.props;
+    const virtualCardsMaxDailyCount = get(collectiveSettings, `virtualCardsMaxDailyCount`) || 100;
     return (
-      <Flex justifyContent="center" mt={4} mb={3}>
-        <H3 mr="1em">
-          <FormattedMessage id="virtualCards.create.number" defaultMessage="Number of gift cards" />
-        </H3>
-        <StyledInput
-          id="virtualcard-numberOfVirtualCards"
-          type="number"
-          step="1"
-          min="1"
-          max="100000"
-          maxWidth="6.5em"
-          onChange={e => this.onChange('numberOfVirtualCards', e.target.value)}
-          value={this.state.values.numberOfVirtualCards}
-          disabled={this.state.submitting}
-        />
-      </Flex>
+      <Container display="flex" flexDirection="column" width={1} justifyContent="center">
+        <P>
+          <FormattedMessage
+            id="VirtualCard.Limitinfo"
+            defaultMessage="Your account is currently limited to {limit} gift cards / day. If you want to increase that limit, please contact {email}."
+            values={{
+              email: <a href={`mailto:support@opencollective.com`}>{'support@opencollective.com'}</a>,
+              limit: get(collectiveSettings, `virtualCardsMaxDailyCount`) || 100,
```

Comment:
We can re-use the variable here

```suggestion
              limit: virtualCardsMaxDailyCount,
```